src/org/labkey/targetedms/view/passport/ProteinWebPart.jsp

```diff
@@ -0,0 +1,251 @@
+<%@ page import="org.labkey.api.settings.AppProps" %>
+<%@ page import="org.labkey.api.view.ActionURL" %>
+<%@ page import="org.labkey.api.view.HttpView" %>
+<%@ page import="org.labkey.api.view.JspView" %>
+<%@ page import="org.labkey.api.view.ViewContext" %>
+<%@ page import="org.labkey.targetedms.model.passport.IKeyword" %>
+<%@ page import="org.labkey.targetedms.model.passport.IProtein" %>
+<%@ page import="java.text.SimpleDateFormat" %>
+<%@ page import="java.util.ArrayList" %>
+<%@ page import="java.util.List" %>
+<%@ page import="org.labkey.api.view.template.ClientDependencies" %>
+<%@ page import="org.labkey.targetedms.TargetedMSController" %>
+<%@ taglib prefix="labkey" uri="http://www.labkey.org/taglib" %>
+
+<%@ page extends="org.labkey.api.jsp.JspBase" %>
+
+<link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+<script src="//code.jquery.com/ui/1.11.4/jquery-ui.js" type="text/javascript"></script>
+
+<%!
+    @Override
+    public void addClientDependencies(ClientDependencies dependencies)
+    {
+        dependencies.add("passport/js/d3.v3.js");
+        dependencies.add("passport/js/util.js");
+        dependencies.add("passport/js/settings.js");
+        dependencies.add("passport/js/protein.js");
+        dependencies.add("passport/js/peakareachart.js");
+        dependencies.add("passport/js/project.js");
+        dependencies.add("passport/js/proteinbar.js");
+        dependencies.add("internal/jQuery");
+
+    }
+%>
+
+<%
+    JspView<?> me = (JspView<?>) HttpView.currentView();
+    IProtein protein = (IProtein)me.getModelBean();
+    final String contextPath = AppProps.getInstance().getContextPath();
+    if(protein == null) {%>
+        Sorry the page you are looking for could not be found.
+    <%} else {%>
+<!--START IMPORTS-->
+
+<script type="text/javascript">
+    LABKEY.requiresCss("passport/css/protein.css");
+    LABKEY.requiresCss("passport/css/peakareachart.css");
+</script>
+<script type="text/javascript">
+    var proteinJSON = <%=unsafe(protein.getJSON().toString(2))%>
+    document.addEventListener("DOMContentLoaded", function() {
+        protein.initialize();
+    });
+    var chomatogramUrl = "<%=h(new ActionURL(TargetedMSController.PrecursorChromatogramChartAction.class, getContainer()))%>";
+    var showPeptideUrl = "<%=h(new ActionURL(TargetedMSController.ShowPeptideAction.class, getContainer()))%>";
+</script>
+<!--END IMPORTS-->
+
+<%%>
+<!-- PROTEIN INFO HEADER START -->
+<div id="passportContainer">
+    <div id="basicproteininfo">
+        <h2 id="proteinName"><%=h(protein.getName())%>
+            <a href="<%=h(new ActionURL(TargetedMSController.DownloadDocumentAction.class, getContainer()))%>id=<%=h(protein.getFile().getRunId())%>">
+                <img src="<%=h(contextPath)%>/passport/img/download.png" style="width:30x; height:30px; margin-left:5px;" alt="Download Skyline dataset" title="Download Skyline dataset from PanoramaWeb.org">
```

Comment:
Omit "from PanoramaWeb.org" since this will be deployed on lots of servers

Reply:
done